deepchecks/utils/abstracts/confusion_matrix_abstract.py

```diff
@@ -9,6 +9,7 @@
 # ----------------------------------------------------------------------------
 #
 """The confusion_matrix_report check module."""
+import textwrap
```

Comment:
What is this used for?
Try to see if you can work without it (we try to limit as much as possible number of external packages)

Reply:
Done